README.md

```diff
@@ -33,4 +33,62 @@ The number of vowels and consonants of screen_name and name is counted, and it i
+
+For illustrating how the heuristic works, let's take for example the user CESARSANCHEZ533.
+
+Assume that he's the user being analyzed.
+First, the interactions and users interacted by him are fetched, using the 'NetworkAnalyzer' class from the 'network_analysis' module(3). The results are better if the synthesis of the interactions of a user is ordered decreasingly by the number of interactions started with them, so its iterated in this order.
+Also the synthesis of the interacted users with the bot_detector_pbb of each one of them is fetched.
+The exact next steps depends on the specific method of the heuristic, and are described below. For now suffice it say that a score is calculated, and if it trespassess a Threshold, then the heuristic returns 1. If not, it returns 0.
+
+There are two approaches and four different methods for computing the value of the heuristic -the score-. All of them implemented, and one can switch between one and another by setting the constant FAKE_PROMOTER_HEUR (in the method 'compute_bot_probability') to the number corresponding to the desired heuristic.
+
+##### Approach Number 1
+
+The main idea of this approach is to compute the number of interactions with users with a bot_detector_pbb higher than some minimum value (BOT_DET_PBB_THRS), and what fraction of the total number of interactions it represents.
+
+###### Method 0
+
+'Filter' the accounts by its bot_detector_pbb, selecting only those that are greater than a minimum (BOT_DET_PBB_THRS).
+Compute the total number of interactions started by the analyzed user (in the example, CESARSANCHEZ533) with the filtered users.
+Evaluate if this number is greater than some threshold(4) (SCORE_TOP_INTRCTNS_THRESHOLD), and return 1 if it is. Otherwise return 0.
+Then divide over the total number of interactions of the user to get the percentage of the total that these interactions represents, and evaluate them against another threshold, and do the same as before.
+
+##### Approach Number 2
+Compute the average bot_detector_pbbs of the users interacted (or interacted users) by the user being analyzed. The methods differ in what they use as weights for the averages.
+
+###### Method 1
+Compute the average bot_detector_pbb of the interacted users. Evaluate against some threshold considered normal. If it's greater, return 1, else return 0.
```

Comment:
Cual threshold?

Debería ser *Compute the average bot_detector_pbb of the interacted users. If the average is greater than XXX threshold, return 1 otherwise 0*